app/k9mail/build.gradle

```diff
@@ -62,15 +62,19 @@ android {
         buildConfig true
     }
 
-    signingConfigs {
-        release
+    def releaseSigning = signingConfigs.debug
```

Comment:
Why?

Reply:
Use the default release signing by default, if not will make the debug signing as the fallback.

Reply:
ed578a6c6348fb9ff7e9cc78a2a771b5f06b3f89